api/lib/infrastructure/repositories/assessment-result-repository.js

```diff
@@ -10,6 +10,10 @@ const CompetenceMark = require('../../domain/models/CompetenceMark');
 function _toDomain({ assessmentResultDTO, competencesMarksDTO }) {
   const competenceMarks = competencesMarksDTO.map((competenceMark) => new CompetenceMark(competenceMark));
 
+  // TODO : Aceol ! You can do better than this !
+  const cleanReproducibilityRate = assessmentResultDTO.reproducibilityRate
+    ? _.toNumber(assessmentResultDTO.reproducibilityRate)
+    : null;
```

Comment:
J'ai pas beaucoup mieux, je vais continuer d'y reflechir...
(dans le code existant, si `assessmentResultDTO.reproducibilityRate` vaut 0,  `cleanReproducibilityRate` vaudra 0)
```suggestion
const reproducibilityRate = assessmentResultDTO.reproducibilityRate ?? null;
const cleanReproducibilityRate = reproducibilityRate !== null ? _.toNumber(reproducibilityRate);
```